apps/arena/lib/arena/game_updater.ex

```diff
@@ -1552,24 +1552,15 @@ defmodule Arena.GameUpdater do
       name: "Circle 1"
     }
 
-    entities_to_collide_with =
+    static_obstacles =
```

Comment:
Shouldn't we add active dynamic obstacles here?

Reply:
This was some creative freedom from our part, there could be some risk in taking power ups in spikes about to go up, if you think this looks more like a bug than a feature i could change it

Reply:
Maybe I wasn't clear enough. I think we should add the dynamic obstacles that are ACTIVE (you can't pass through) so the liftable entity can't go through as well.
It's ok to ignore it if the obstacle is off (underground).

Reply:
oooh okey, fair enough

Reply:
You forgot to change the variable name

Reply:
Whoops, fixed